assets/src/components/amp-support/notices.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import {
+	AMPNotice,
+	NOTICE_SIZE_SMALL,
+	NOTICE_TYPE_INFO,
+	NOTICE_TYPE_SUCCESS,
+	NOTICE_TYPE_WARNING,
+} from '../amp-notice';
+
+/**
+ * Render notices on site support page.
+ *
+ * @param {Object} props Component props.
+ * @return {JSX.Element|null} HTML markup for notice data.
+ */
+export function Notices( props ) {
+	const { data, args, ampValidatedPostCount } = props;
```

Comment:
Would it make sense to name the component in a more specific way, e.g. `ValidationResultsNotice`?

Also, we're not using `props` anywhere in the component so we could destructure it directly inside the function declaration:

```suggestion
export function ValidationResultsNotice( { data, args, ampValidatedPostCount } ) {
```